addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -389,9 +393,12 @@ def _invoice_constraints_peppol_en16931_ubl(self, invoice, vals):
                 # [NL-R-003] For suppliers in the Netherlands, the legal entity identifier MUST be either a
                 # KVK or OIN number (schemeID 0106 or 0190)
                 'nl_r_003': _(
-                    "%s should have a KVK or OIN number: the Peppol e-address (EAS) should be '0106' or '0190'.",
+                     "%s should have a KVK or OIN number set in Company ID field or as Peppol e-address (EAS code 0106 or 0190).",
```

Comment:
```suggestion
                    "%s should have a KVK or OIN number set in Company ID field or as Peppol e-address (EAS code 0106 or 0190).",
```
Looks like you introduced a 5th space there :)